internal/page/certificateprovider/register.go

```diff
@@ -156,6 +159,8 @@ func Register(
 		ProvideCertificate(tmpls.Get("provide_certificate.gohtml"), lpaStoreResolvingService, certificateProviderStore, notifyClient, shareCodeSender, lpaStoreClient, time.Now))
 	handleCertificateProvider(page.Paths.CertificateProvider.CertificateProvided, page.None,
 		Guidance(tmpls.Get("certificate_provided.gohtml"), lpaStoreResolvingService, certificateProviderStore))
+	handleCertificateProvider(page.Paths.CertificateProvider.ConfirmDontWantToBeCertificateProvider, page.None,
```

Comment:
maybe this page should have a back button?

Reply:
or if not it needs a secondary button, as at the moment you only have the choice of "Confirm"

Reply:
We'll go with a back button for now and raise in the next review on if a secondary button is more appropriate.